src/components/IconButton/IconButton.jsx

```diff
@@ -44,9 +44,9 @@ const IconButton = forwardRef(
     const iconSize = useMemo(() => {
       if (size === IconButton.sizes.XXS) return 16;
       if (size === IconButton.sizes.XS) return 16;
-      if (size === IconButton.sizes.SMALL) return 24;
-      if (size === IconButton.sizes.MEDIUM) return 24;
-      if (size === IconButton.sizes.LARGE) return 24;
+      if (size === IconButton.sizes.SMALL) return 20;
+      if (size === IconButton.sizes.MEDIUM) return 20;
+      if (size === IconButton.sizes.LARGE) return 20;
       return 24;
```

Comment:
Consider using a switch-case:
```js
switch (size) {
  case IconButton.sizes.XXS:
  case IconButton.sizes.XS:
    return 16;
  case IconButton.sizes.SMALL:
  case IconButton.sizes.MEDIUM:
  case IconButton.sizes.LARGE:
    return 20;
  default:
    return 24;
}
```

Reply:
it should be just a quick fix but I guess we can refactor it a bit